researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/ResearchStackDAO.java

```diff
@@ -0,0 +1,28 @@
+package org.sagebionetworks.bridge.researchstack;
+
+import android.content.Context;
+
+import org.researchstack.backbone.model.User;
+import org.sagebionetworks.bridge.android.manager.dao.SharedPreferencesJsonDAO;
+
+/**
+ * Created by jyliu on 2/9/2017.
+ */
+
+public class ResearchStackDAO extends SharedPreferencesJsonDAO {
```

Comment:
What's the difference between ResearchStackDAO which contains User, and the AccountDAO? They seem like they are duplicated in some ways.